configure.ac

```diff
@@ -533,6 +533,9 @@ AC_ARG_WITH([coverage],
 yes_no_try "$with_coverage" "no"
 with_coverage=$?
 
+AC_DEFINE_UNQUOTED([BUILD_COVERAGE], [$with_coverage], [Build with code coverage])
+AM_CONDITIONAL([BUILD_COVERAGE], [test $with_coverage -ne $DISABLED])
```

Comment:
Let's call this `PCMK__WITH_COVERAGE`

Reply:
Both, or just the AM_CONDITIONAL one?

Reply:
Only the C one is important, the other can be either way